app/com/linkedin/drelephant/analysis/AnalyticJobGeneratorHadoop2.java

```diff
@@ -198,7 +198,7 @@ private void updateAuthToken() {
    */
   private JsonNode readJsonNode(URL url)
       throws IOException, AuthenticationException {
-    HttpURLConnection conn = _authenticatedURL.openConnection(url, _token);
+//    HttpURLConnection conn = _authenticatedURL.openConnection(url, _token);
```

Comment:
Would this work fine for Kerberos enabled cluster?

Reply:
Yes it works. Its being used on a Production Cluster